src/index.html

```diff
@@ -11,6 +11,32 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="margin-form"
```

Comment:
why is it `margin-form` ?

Reply:
changed to .body__formBig__margins
Because, if i will install individually margins for big and small searchbars, like margin-top: 20px, the picture will be the same, but when i'm making "npm test" - tests are not allowing me to complete, because passing only 1/6 tests. but if i have separate class for big searchbar in selector "form" - all is fine